### ISSUE TITLE
token_type_ids 관련 이슈

### ISSUE THREAD

**Issue by Yebin46** [SKTBrain/KoBERT#66]
안녕하세요. 먼저 좋은 소스 공유해주셔서 진심으로 감사드립니다.

다름이 아니라 token_type_ids가 제대로 만들어지지 않는 것 같아 문의드립니다.
제가 사용한 코드는 아래와 같습니다. (transformers 라이브러리를 사용했습니다)

~~~ python
import torch
from transformers import (
    BertModel, BertForMaskedLM, DataCollatorForLanguageModeling
)
from kobert_tokenizer import KoBERTTokenizer
from torch.utils.data import DataLoader

tokenizer = KoBERTTokenizer.from_pretrained('skt/kobert-base-v1')
model = BertForMaskedLM.from_pretrained('skt/kobert-base-v1')

data_collator = DataCollatorForLanguageModeling(tokenizer=tokenizer)
train_dataloader = DataLoader(
    train_dataset, shuffle=True, batch_size=8, collate_fn=data_collator
)

~~~

Dataloader로부터 생성된 batch를 확인해보니 padding된 부분의 token_type_ids가 3으로 생성되는 것을 확인할 수 있었습니다.
그리고 이 때문에 모델에 token_type_ids를 input으로 넣었을 때 에러가 발생하였습니다.
![image](https://user-images.githubusercontent.com/68219137/138621981-e2f9819d-a682-43a0-be00-a50386a56f87.png)

혹시 제가 잘못한/잘못 생각한 부분이 있다면 말씀 부탁드립니다.
감사합니다.

**Comment by haven-jeon** [comment]
별다른 언급이 없어 해당 이슈는 클로즈 합니다. 필요시 열어주세요.
